### PR TITLE
uInverter revB

### LINKED ISSUE #209: Remove uInverter Docs folder
We have migrated the uInverter docs to `docs.amdc.dev`. It is time to confirm that all pertinent info has been brought over and to remove the `docs` folder in the `uInverter` accessory board folder.

### PR DESCRIPTION
**Pull request by purohitparth25**
Branch for uInverter revB development. 

For details on modifications done in revB, check these approved PRs:
- [uInverter revB Schematic review](https://github.com/Severson-Group/AMDC-Hardware/pull/198)
- [uInverter revB PCB review](https://github.com/Severson-Group/AMDC-Hardware/pull/203)

For `CHANGELOG`, see [`CHANGELOG.MD`](https://github.com/Severson-Group/AMDC-Hardware/blob/uInverter_revB/Accessories/ExpansionBoard_uInverter/CHANGELOG.md)

### CODE REVIEW

**Comment by purohitparth25** [comment]
The request changes comment from @elsevers did not update here it seems. After previous review, I have implemented following changes in this PR:
- Corrected the note about heatsink slot on revB landing page
- Updated CHANGELOG as per @npetersen2 's suggestions
- Added gerbers for revB
- Deleted the docs folder

This PR is ready to be reviewed now!

File: Accessories/ExpansionBoard_uInverter/REV20210506A/uInverter-rev-a_bom.csv [hunk 1]
```diff
@@ -1,20 +1,20 @@
 Supplier Part Number 1,Quantity,Comment,Description,Designator,Footprint,LibRef
```
Comment (npetersen2) [inline Accessories/ExpansionBoard_uInverter/REV20210506A/uInverter-rev-a_bom.csv:1]: Is there a reason the REV A BOM is changed in this PR?

File: Accessories/ExpansionBoard_uInverter/REV20220329B/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+| Thickness         | 1.7mm         |
+| Min Track/Spacing | 6/6mil          |
+| Min Hole Size     | 14mil           |
+| Silkscreen        | Top Only      |
+| Part Count        | 66            |
+
+
+## Connectors and connections
+
+Please check the [docs folder](../docs/) for more details on the connectors and pin-outs and other documentation.
```
Comment (npetersen2) [inline Accessories/ExpansionBoard_uInverter/REV20220329B/README.md]: I believe this PR removes the `docs/` folder.

Please update this text to point the user to the `docs.amdc.dev` website for the uInverter section

File: Accessories/ExpansionBoard_uInverter/REV20220329B/gerbers/PCB_uinv-RectHoles.TXT [hunk 3]
```diff
@@ -0,0 +1,20 @@
+M48
```
Comment (npetersen2) [inline Accessories/ExpansionBoard_uInverter/REV20220329B/gerbers/PCB_uinv-RectHoles.TXT:1]: Is it standard to include a file for rectangular holes like this?

I do not have experience specifying slots and rect holes, so I do not know.

@GnomeTek thoughts?

File: Accessories/ExpansionBoard_uInverter/REV20220329B/uInv_revB_BOM.csv [hunk 4]
```diff
@@ -0,0 +1,21 @@
+Supplier Part Number 1,Quantity,Comment,Description,Designator,Footprint,LibRef
+
+"399-17618-1-ND","3","0.1uF","Cap Ceramic 0.1uF 35V X7R 10% SMD 0805","C1A, C1B, C1C","0805_CAP","C0805C104K6RACTU"
+"490-10514-2-ND","5","10uF","0805 35 V 10 uF �10% Tolerance X5R Multilayer Ceramic Capacitor","C2A, C2B, C2C, C5, C6","0805_CAP","GRM21BR6YA106KE43L"
+"445-7551-1-ND","6","2.2uF","2.2 �F �10% 35V Ceramic Capacitor X7R 0805","C3A, C3B, C3C, C4A, C4B, C4C","0805_CAP","C2012X7R1V225K125AB"
```
Comment (npetersen2) [inline Accessories/ExpansionBoard_uInverter/REV20220329B/uInv_revB_BOM.csv]: Can you remove the "funny" text characters from the BOM?

image

File: Accessories/ExpansionBoard_uInverter/REV20220329B/uInv_revB_BOM.csv [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+"553-3011-1-ND","3","100u","FIXED IND 100UH 5.3A 118 MOHM","L1A, L1B, L1C","FP-PA4344_104NLT-MFG","PA4344.104NLT"
+"36-5005-ND","15","Test Point","Conn Test Point SKT 1 POS ST Thru-Hole 1 Port","PWM-A, PWM-B, PWM-C, TP1A, TP1B, TP1C, TP2A, TP2B, TP2C, TP3A, TP3B, TP3C, TP4, TP5, TP6","TP_1.6MM","Test Point 36-5005ND"
+"A97581-ND","3","5-1634503-1","CONN BNC JACK STR 50 OHM PCB","Q1A, Q1B, Q1C","5-1634503-1","5-1634503-1"
+"541-1.00CCCT-ND","3","1R","Res Thick Film 0805 1 Ohm 1% 1/8W","R1A, R1B, R1C","0805_RES","CRCW08051R00FKEA"
+"541-2841-1-ND","2","2.2K","Res Thick Film 0805 2.2K Ohm 1% 0.4W","R2, R3","0805_RES","RCS08052K20FKEA"
+"RHM1.10KAECT-ND","2","1.1K","Res Thick Film 0805 1.1K Ohm 1% 0.4W","R4, R5","0805_RES","ESR10EZPF1101"
+"15FR150E-ND","3","150m","Resistor; Wirewound; Res 0.15 Ohms; Pwr-Rtg 5 W; Tol 1%; Axial; Silicone; Cur-Sense","Rsense1A, Rsense1B, Rsense1C","RES_15_OHM","15FR150E"
+"IXDN614CI-ND","3","IXDN614CI","14-Ampere Low-Side Ultrafast MOSFET Driver 5-Lead TO-220","U1A, U1B, U1C","IXYS_TO-220-5_IXY","IXDN614CI"
+"INA143U-ND","3","INA143U","SP Amp Difference Amplifier Single �18V/36V 8-Pin SOIC Tube","U2A, U2B, U2C","SOIC127P599X175-8N","INA143U"
```
Comment (npetersen2) [inline Accessories/ExpansionBoard_uInverter/REV20220329B/uInv_revB_BOM.csv]: Can you remove the "funny" text characters from the BOM?

image

File: Accessories/ExpansionBoard_uInverter/REV20220329B/README.md (hunk 2 above)
Comment (npetersen2) [reply Accessories/ExpansionBoard_uInverter/REV20220329B/README.md]: Also propagate this change to the `REV A` folder readme file

**Review by npetersen2 — CHANGES_REQUESTED** [review]
Thanks @purohitparth25 ! See my inline comments.

Also, I submitted your gerber files to an online viewer to ensure they look good, but it is rendering them odd. Can you confirm:
- [x] the units between drill files and gerbers are consistent?
- [x] the board outline is on Mechanical Layer 1?

image

File: Accessories/ExpansionBoard_uInverter/REV20210506A/uInverter-rev-a_bom.csv (hunk 1 above)
Comment (purohitparth25) [reply Accessories/ExpansionBoard_uInverter/REV20210506A/uInverter-rev-a_bom.csv:1]: Yes, the description in docs and the BOM itself mentions a 2.2K resistor, however the part number mentioned was for 2.49K, so I corrected that and replaced it with a 2.2K part no. (same 0603 size)

File: Accessories/ExpansionBoard_uInverter/REV20220329B/README.md (hunk 2 above)
Comment (purohitparth25) [reply Accessories/ExpansionBoard_uInverter/REV20220329B/README.md]: I updated the links to point to the docs.amdc.dev website's uInverter section.

File: Accessories/ExpansionBoard_uInverter/REV20220329B/uInv_revB_BOM.csv (hunk 4 above)
Comment (purohitparth25) [reply Accessories/ExpansionBoard_uInverter/REV20220329B/uInv_revB_BOM.csv]: Removed.

File: Accessories/ExpansionBoard_uInverter/REV20220329B/uInv_revB_BOM.csv (hunk 5 above)
Comment (purohitparth25) [reply Accessories/ExpansionBoard_uInverter/REV20220329B/uInv_revB_BOM.csv]: Removed.

File: Accessories/ExpansionBoard_uInverter/REV20210506A/uInverter-rev-a_bom.csv (hunk 1 above)
Comment (purohitparth25) [reply Accessories/ExpansionBoard_uInverter/REV20210506A/uInverter-rev-a_bom.csv:1]: 1. Units for both, drill files and Gerber files are inches
2. I did not add the board outline before. Now, I have added the outline on `keepout layer` and pushed the changes.

File: Accessories/ExpansionBoard_uInverter/REV20220329B/gerbers/PCB_uinv-RectHoles.TXT (hunk 3 above)
Comment (purohitparth25) [reply Accessories/ExpansionBoard_uInverter/REV20220329B/gerbers/PCB_uinv-RectHoles.TXT:1]: I have pushed both drill files for now.

**Comment by elsevers** [comment]
@purohitparth25, please be sure to re-request @npetersen2's review when you are ready.

File: Accessories/ExpansionBoard_uInverter/REV20210506A/uInverter-rev-a_bom.csv (hunk 1 above)
Comment (npetersen2) [reply Accessories/ExpansionBoard_uInverter/REV20210506A/uInverter-rev-a_bom.csv:1]: @purohitparth25 Sounds good for the REV A BOM change, I trust this change was propagated to the REV B BOM as well.

**Review by npetersen2 — CHANGES_REQUESTED** [review]
@purohitparth25 thanks for the updates.

I uploaded the gerbers to oshpark to render them again, it looks pretty good, except I noticed the silkscreen revision label has not been updated to the REV B string.

Please update it, regenerate the gerbers, and rerequest review. I think with that change, it will be ready to go!

image

File: Accessories/ExpansionBoard_uInverter/REV20210506A/uInverter-rev-a_bom.csv (hunk 1 above)
Comment (purohitparth25) [reply Accessories/ExpansionBoard_uInverter/REV20210506A/uInverter-rev-a_bom.csv:1]: @npetersen2, we are using an altogether different part number in revB (0603 size as compared to 0805 in revA)

Comment (npetersen2) [reply Accessories/ExpansionBoard_uInverter/REV20210506A/uInverter-rev-a_bom.csv:1]: @purohitparth25 Excellent, thanks!

**Comment by purohitparth25** [comment]
@npetersen2 , I changed the silkscreen string to revB.

image

**Review by npetersen2 — APPROVED** [review]
@purohitparth25 nice work on this board revision!

I am looking forward to seeing how well these changes improve the performance :)

**Comment by npetersen2** [comment]
**Please be sure to `Squash and merge` this PR into `develop` branch**

**Review by elsevers — APPROVED** [review]
Looks good, thanks @purohitparth25 !

**Comment by elsevers** [comment]
Adding this document that @purohitparth25  created on testing out the revB design for archival purposes.
[uInverter_revB.pdf](https://github.com/Severson-Group/AMDC-Hardware/files/9480978/uInverter_revB.pdf)

**Pull request merged**